protocols/kad/src/protocol.rs

```diff
@@ -165,6 +170,16 @@ impl ProtocolConfig {
     pub fn set_max_packet_size(&mut self, size: usize) {
         self.max_packet_size = size;
     }
+
+    /// Modifies outbount_substreams timeout.
+    pub fn set_outbound_substreams_timeout(&mut self, value: u64) {
+        self.outbound_substreams_timeout_s = value;
+    }
+
+    /// Getter of outbount_substreams_timeout_s.
+    pub fn outbound_substreams_timeout_s(&self) -> u64 {
+        self.outbound_substreams_timeout_s
+    }
```

Comment:
We also need to add these to `behavior::Config` because that's the user-facing config structure that wraps `ProtocolConfig`.

Reply:
> We also need to add `set_outbound_substreams_timeout` to `behavior::Config`

PR updated to add it in